docs/src/pages/_app.tsx

```diff
@@ -79,12 +86,25 @@ const MyApp = ({ Component, pageProps }: AppProps) => {
                   {pageProps.caption && (
                     <Text variant="page-caption">{pageProps.caption}</Text>
                   )}
-                  {pageProps?.switchTheme && <ThemeSelect />}
                 </Header>
               )}
               <Box css={{ display: 'flex', gap: 'large-2' }}>
                 <Container contentType="content" size="large">
-                  {pageProps?.figma && <FigmaLink href={pageProps.figma} />}
+                  <Inline>
```

Comment:
Can we give the links some margin-bottom? Because when we have no theme switch theres no spaces to the content.